.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode12
+osx_image: xcode12.2
```

Comment:
What does this mean? The Xcode version that Travis builds with?

Reply:
Yes, There was a compiling  issue with Xcode 12 and updated to 12.2